src/queries/vote/index.ts

```diff
@@ -165,3 +166,31 @@ export const useAdditionalInfoById = (id?: number) => {
 
   return items?.find(item => item.id === id);
 };
+
+export const useHasAccountActiveProposal = () => {
+  const { account } = useIconReact();
+  const { data: platformDay } = usePlatformDayQuery();
+
+  return useQuery<boolean | undefined>(QUERY_KEYS.Vote.IsAnyProposalActive(account || ''), async () => {
+    if (account) {
+      const proposals = await bnJs.Governance.getProposals(1, 100);
+      return (
+        (await proposals
```

Comment:
Did you test the logic? I think it won't work. Because I think the `filter` function doesn't support an async function as a predicate.

Reply:
I did and it accidentally worked :D but you were right, it wasn't functioning properly, thanks for pointing that out! 
I've moved the async part to map function.